README.md

```diff
@@ -107,9 +107,12 @@ See the [Unit test documentation](Source/UnitTests/README.md)
 The current version of Boogie is noted in a [build property](Source/Directory.Build.props).
 To push a new version to nuget, perform the following steps:
 
-- Update the version (e.g., x.y.z) in `Source/Directory.Build.props` and commit the change locally
-- `git tag vx.y.z`
-- `git push origin master vx.y.z` where `origin` denotes the remote on github.com
+- Update the version (e.g., x.y.z) in `Source/Directory.Build.props`
+- `git add Source/Directory.Build.props; git commit -m "Release version x.y.z"` to commit the change locally
+- `git tag vx.y.z` to create a local tag for the version
+- `git push origin master:release-vx.y.z` to push your change in a separate branch, where `origin` normally denotes the remote on github.com
```

Comment:
I would prefer using forks for pushing branches, instead of the origin.

Reply:
One can definitely use forks to push branches if they prefer - that's the beauty of git - but consider this.

Forks were invented so that [external contributors](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) could create their own versions of open-source software and have the right to push commits without affecting the original repository structure, until possibly the forks becomes a pull request.

Versioning is something that maintainers have to perform and it shouldn't be delegated to forks.
Moreover, it is common for people to disable pushing tags on forks by default, because tags make sense only be used in the "upstream" repository.

Reply:
Alright :)